modules/ossm-control-plane-deploy.adoc

```diff
@@ -84,59 +79,57 @@ Follow this procedure to deploy the {ProductName} control plane the command line
-----
 +
-. Execute the following command to see the status of the control plane installation.
+. Create a `ServiceMeshControlPlane` file named `istio-installation.yaml` using the following example in "Customize the {ProductName} installation". For production deployments you _must_ customize the default link:https://github.com/maistra/istio-operator/blob/maistra-2.0/deploy/examples/maistra_v2_servicemeshcontrolplane_cr_auth.yaml[template].
 +
-[source,terminal]
+.Example version 2.0 istio-installation.yaml
+[source,yaml]
 ----
-$ oc get smcp -n istio-system
+apiVersion: maistra.io/v2
+kind: ServiceMeshControlPlane
+metadata:
+  name: basic
+  namespace: openshift-operators
+spec:
+  version: v2.0
+  tracing:
+    type: Jaeger
+    sampling: 10000
+  addons:
+    jaeger:
+      install:
+        storage:
+          type: Memory
+    kiali:
+      enabled: true
+    grafana:
+      enabled: true
 ----
 +
-The installation has finished successfully when the READY column is true.
+. Run the following command to deploy the control plane, where `<istio_installation.yaml>` includes the full path to your file.
 +
+[source,terminal]
 ----
-NAME           READY
-basic-install   True
+$ oc create -n istio-system -f <istio_installation.yaml>
 ----
 +
-. Run the following command to watch the progress of the Pods during the installation process:
+. Execute the following command to see the status of the control plane installation.
 +
+[source,terminal]
 ----
-$ oc get pods -n istio-system -w
+$ oc get smcp -n istio-system
 ----
 +
-You should see output similar to the following:
+The installation has finished successfully when the `STATUS` column is `ComponentsReady`.
 +
-.Example output
-[source,terminal]
 ----
-NAME                                     READY   STATUS             RESTARTS   AGE
-grafana-7bf5764d9d-2b2f6                 2/2     Running            0          28h
-istio-citadel-576b9c5bbd-z84z4           1/1     Running            0          28h
-istio-egressgateway-5476bc4656-r4zdv     1/1     Running            0          28h
-istio-galley-7d57b47bb7-lqdxv            1/1     Running            0          28h
-istio-ingressgateway-dbb8f7f46-ct6n5     1/1     Running            0          28h
-istio-pilot-546bf69578-ccg5x             2/2     Running            0          28h
-istio-policy-77fd498655-7pvjw            2/2     Running            0          28h
-istio-sidecar-injector-df45bd899-ctxdt   1/1     Running            0          28h
-istio-telemetry-66f697d6d5-cj28l         2/2     Running            0          28h
-jaeger-896945cbc-7lqrr                   2/2     Running            0          11h
-kiali-78d9c5b87c-snjzh                   1/1     Running            0          22h
-prometheus-6dff867c97-gr2n5              2/2     Running            0          28h
+NAME            READY   STATUS            PROFILES      VERSION   AGE     IMAGE REGISTRY
+basic-install   9/9     ComponentsReady   ["default"]   2.0.0     3m31s   
```

Comment:
If the resource's name was `basic`, this will be `basic` instead of `basic-install`